Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/TestAutonomous.java

```diff
@@ -0,0 +1,41 @@
+package org.firstinspires.ftc.team417_PowerPlay;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.team417_PowerPlay.drive.SampleMecanumDrive;
+
+@Autonomous (name="strafe test")
+public class TestAutonomous extends LinearOpMode {
+
+    // Coordinates for various tiles referenced on page 46 of:
+    // https://www.firstinspires.org/sites/default/files/uploads/resource_library/ftc/game-manual-part-2-traditional.pdf
+    // Robot's position starts at (0, 0) - all tile coordinates are in relation to this starting position
+    private static final double Y_POSITION_A = -30;
+    private static final double X_POSITION_C = 41;
```

Comment:
I still don't understand what this means tbh.

Reply:
Basically the convention for identifying the field tiles are labeling the columns with letters going left to right, and the rows with numbers from the bottom to the top (looking at the field from a top down view with the audience side at the bottom and the red alliance side on the right). Based on this we just decided to declare constants for each column or row tile we will travel to, but I am now just realizing that the second one should actually be X_POSITION_1 instead of C which I will also change